cmd/agent/local/local.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 	"github.com/crossplane/crossplane/apis/apiextensions"
 
-	"github.com/crossplane/agent/pkg/controllers/publication"
+	"github.com/crossplane/agent/pkg/controllers/xrd"
```

Comment:
The controller for the corresponding type upstream is named `definition` - I think we should be consistent here. I'm happy if we rename both controller packages to `xrd`, though.

Reply:
In this repo, there is a lot of operations going on with both CRDs and XRDs so I'm hesitant to call any of them `definition`. In fact, we have `pkg/controllers/crd` and `pkg/controllers/xrd` packages. So, I'd opt for `xrd` at least for this repo.

Reply:
> Is that not also true of the c/c apiextensions controllers?

Do you mean that the `apiextensions` controllers in the upstream could also have that separation? In this case, the `pkg/controllers/crd` actually watches the type `CustomResourceDefinition`. I don't think we have a controller in c/c that watches for `CustomResourceDefinition`s.